hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestColumnStatsIndex.scala

```diff
@@ -847,7 +856,7 @@ class TestColumnStatsIndex extends ColumnStatIndexTestBase {
       .mode(SaveMode.Overwrite)
       .save(basePath)
 
-    metaClient = HoodieTableMetaClient.reload(metaClient)
+    metaClient = HoodieTableMetaClient.builder().setBasePath(basePath).setConf(storageConf).build()
```

Comment:
why does the meta client reload not work now?

Reply:
The new table configs are not reflected with reload. Reload takes some arguments from old meta client (created in harness utility class I think). Those properties do not work here.

Reply:
Maybe we just copy the tests instead of modification on existing one, the additional version param makes the existing tests much more complicated and hard to maintain.